src/gsy_e/models/strategy/external_strategies/storage.py

```diff
@@ -441,7 +441,7 @@ def event_tick(self) -> None:
         if not self.connected and not self.is_aggregator_controlled:
             super().event_tick()
         else:
-            self.state.tick(self.area, self.spot_market.time_slot)
+            self.state.check_state(self.spot_market.time_slot)
```

Comment:
Thanks for maintaining this! It is really more or less a failsafe/sanity check, but has been very useful for detecting issues over the years